openpgp/packet/public_key.go

```diff
@@ -195,6 +256,7 @@ func (pk *PublicKey) parse(r io.Reader) (err error) {
 	}
 	pk.CreationTime = time.Unix(int64(uint32(buf[1])<<24|uint32(buf[2])<<16|uint32(buf[3])<<8|uint32(buf[4])), 0)
 	pk.PubKeyAlgo = PublicKeyAlgorithm(buf[5])
+	// Ignore four-ocet 
```

Comment:
```suggestion
	// Ignore four-ocet length
```
? Maybe something is missing here?